olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/DataRequest.h

```diff
@@ -49,16 +51,19 @@ class DATASERVICE_READ_API DataRequest final {
    *
    * @return A reference to the updated `DataRequest` instance.
    */
+  OLP_SDK_DEPRECATED("Deprecated, to be removed in 05.2020")
```

Comment:
```suggestion
  OLP_SDK_DEPRECATED("Deprecated, to be removed in 06.2020")
```
Same for below once.